content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -15,7 +15,6 @@ observing them.
 Unlike Pods that are managed by the control plane (eg, a
 {{< glossary_tooltip text="Deployment" term_id="deployment" >}};
 instead, the kubelet watches each static Pod (and restarts it if it crashes).
-There are no health checks for the containers in a static Pod.
```

Comment:
@relyt0925 , Would you also fix a few nits:
- Line 16: There appears to be a missing closing parenthesis; also, could you substitute `for example` for `eg`.
Unlike Pods that are managed by the control plane (for example, a
{{< glossary_tooltip text="Deployment" term_id="deployment" >}}); instead, the kubelet watches each static Pod ...

The kubelet manages the lifecycle of static Pods.

- Line 21, The kubelet vs Kubelet

Reply:
Yep I can!

Reply:
Thanks!
/lgtm